### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlperMumcular**

### CODE REVIEW

**Comment by aysekelleci** [comment]
test

**Comment by aysekelleci** [comment]
test

**Comment by aysekelleci** [comment]
<p>testing</p>

**Comment by aysekelleci** [comment]
<p>test comment</p>

**Comment by AlperMumcular** [comment]
**ACTIVE**: <p>Hello world!</p>

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
 
 ## PDF Encryption
```
Comment (Ece-Kahraman) [inline README.md:6]: question(non-blocking): hi

**Review by Ece-Kahraman — COMMENTED** [review]
yooo

File: README.md (hunk 1 above)
Comment (Ece-Kahraman) [reply README.md:6]: aaaa

**Review by vedxyz — CHANGES_REQUESTED** [review]
:basketball:

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>cdvf</p>

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>good job</p>

File: README.md (hunk 1 above)
Comment (vIremAydin) [inline README.md:6]: none(non-blocking): ömçmömömömömömö

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
...
 
-On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFs:
+On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFsss:
```
Comment (vIremAydin) [inline README.md:8]: zxcxzczxcxzcxzcxzc

**Review by vIremAydin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vIremAydin) [inline README.md:8]: none(non-blocking): kjbjhkhb

**Review by vIremAydin — CHANGES_REQUESTED** [review]

**Comment by vIremAydin** [comment]
**ACTIVE**: <p>sś</p>

**Comment by aysekelleci** [comment]
> 
> **ACTIVE**: <p>csdvv</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2068057983 

dwefrgt

**Comment by aysekelleci** [comment]
> 
> **ACTIVE**: <p>ccsdvv</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2068057983 

deneme

**Comment by aysekelleci** [comment]
> 
> 
> **ACTIVE**: <p>denemeedb</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2069152469 

dkldfjf

**Comment by vIremAydin** [comment]
> **Closed**: <p>Cannot respond to another bot.vdşdv</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2042617736 

https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2042617736

**Comment by HubReview-tester** [comment]
> 
Skipped: Cannot respond to another bot.
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2069841077 

test reply 1

**Comment by HubReview-tester** [comment]
> 
> 
Skipped: Cannot respond to another bot.
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2069841077 

test reply 1
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2072138751 

test reply 2

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
...
```
Comment (aysekelleci) [inline README.md:7]: none(non-blocking): djkwehfkjhr

**Review by aysekelleci — COMMENTED** [review]
comment

File: README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
 
 ## PDF Encryption
 
-On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFs:
```
Comment (vIremAydin) [inline README.md:7]: none(non-blocking): fdvgrfvfgvf

File: README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
```
Comment (vIremAydin) [inline README.md:4]: none(non-blocking): grtgtrrt

File: README.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ Find the project website at [cs49x.hubreview.app](https://cs49x.hubreview.app).
 
 ## PDF Encryption
 
-On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFs:
+On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFsss:
 
 ```sh
 # Replace `[pass]`s with a password
```
Comment (HubReview-tester) [inline README.md:10]: ıssue(blocking): Replace this, may cause security problems

**Review by HubReview-tester — CHANGES_REQUESTED** [review]
please change problematic lines

**Comment by HubReview-tester** [comment]
> **Closed**: <p>Cannot respond to another bot.vdşdv</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/2#issuecomment-2042617736 

meraba

**Pull request closed without merging**